the-api-swagger.yaml

```diff
@@ -0,0 +1,131 @@
+openapi: 3.0.3
```

Comment:
Feels like this would go better under a directory, not in the root of the project.

Needs a better file name.

Reply:
Moved to the folder where it would be searched by the automatic publishing.